### DISCUSSION TITLE
Getting rid of `Invalid @since 6.10 tag on allowConflicts(); expecting @since 7.0` error markers

### DISCUSSION THREAD

**Discussion by p-bakker** [eclipse-jgit/jgit#69] [Q&A] [answered]
Hi,

Am trying to get JGit to compile locally (after switching over from https://git.eclipse.org/r/jgit/jgit > https://github.com/eclipse-jgit/jgit) and updating my master branch to the latest commit (after it had been on a branch off of `stable-6.7`), but having a bunch of errors like  `Invalid @since 6.xx tag on yyy(); expecting @since 7.0`  

Now, I'm sure there's a logical reason and an easy step to resolve all of this, but I'm just not finding it. I had setup my development environment using https://wiki.eclipse.org/EGit/Contributor_Guide#Automated_Developer_Setup in the past and I've been looking at https://github.com/eclipse-egit/egit/wiki/Contributor-Guide for clues, but no dice so far.

Any pointers appreciated!

**Comment by iloveeclipse** [comment, marked as answer]
You have to set API baseline to the exact JGit release that preceeds the current one. E.g. if current is 7.0, API baseline should be set to 6.10.

**Reply by p-bakker** [reply]
Tnx, managed to get it going

In > Preferences > Plug-in Development > API Baselines I just had:
- EGit-6.5.0
- EGit-6.6.0
- EGit-6.7.0 (default baseline) 

I manually added the one I needed. Is that how it should be done? Maybe make a note in the contributing guide?

**Reply by p-bakker** [reply]
Maybe the answer to my question ought to have been in https://github.com/eclipse-egit/egit/wiki/Contributor-Guide#java-requirements , behind the dead link to [Installing Execution Environment Descriptions](https://github.com/eclipse-egit/egit/wiki/Execution_Environments#Installing_Execution_Environment_Descriptions)? (see https://github.com/eclipse-egit/egit/discussions/44)
